core/templates/dev/head/services/ExplorationHtmlFormatterService.js

```diff
@@ -23,8 +23,21 @@ oppia.factory('ExplorationHtmlFormatterService', [
   '$filter', 'extensionTagAssemblerService', 'HtmlEscaperService',
   function($filter, extensionTagAssemblerService, HtmlEscaperService) {
     return {
+      /**
+      * @param {string} interactionId - The id of interaction in camel case.
+      * @param {array} interactionCustomizationArgSpecs - The various attributes
+      * that depends on interaction.
+      * @param {boolean} canAcceptAnswers - Whether or not last answer is
+      * displayed permanently after learner has submitted an answer for
+      * non-inline interactions.
+      * If called from editor tab, set this to false.
```

Comment:
You'll need to be more precise. The solution editor is in the editor tab. Yet you've set it to true in the files above.

Reply:
Oh, they needed to be false, changed that.